lte/gateway/python/magma/mobilityd/ip_allocator_dhcp.py

```diff
@@ -105,6 +111,7 @@ def _monitor_dhcp_state(self):
             with self.dhcp_wait:
                 for dhcp_key, dhcp_desc in self._store.dhcp_store.items():
                     logging.debug("monitor: %s", dhcp_desc)
+                    save_file = f"/tmp/dhcp_cli_{dhcp_desc.mac}.json"
```

Comment:
You could look into the tempfile module to create unique temporary files.

Reply:
Done